### PR TITLE
Add Woodstar experiment

### PR DESCRIPTION
**Pull request by roji**

### CODE REVIEW

**Review by SamMonoRT — APPROVED** [review]

**Review by AndriySvyryd — APPROVED** [review]
🐑  🇮🇹

File: Woodstar/Woodstar.Benchmark/Benchmarks.cs [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+            Add(new SimpleJobAttribute(targetCount: 20).Config);
+            AddDiagnoser(MemoryDiagnoser.Default);
+            AddDiagnoser(ThreadingDiagnoser.Default);
+            AddColumn(new TagColumn("Connections", name => (name.EndsWith("Pipelined") ? 1 : Connections).ToString()));
+        }
+    }
+
+    const string EndPoint = "127.0.0.1:1433";
+    const string Username = "sa";
+    const string Password = "Abcd5678";
```
Comment (AndriySvyryd) [inline Woodstar/Woodstar.Benchmark/Benchmarks.cs:33]: This will avoid potential flagging

```suggestion
    const string Password = "PLACEHOLDER";
```

Comment (roji) [reply Woodstar/Woodstar.Benchmark/Benchmarks.cs:33]: Oops, already merged.

File: Woodstar/Woodstar.FunctionalTests/DatabaseService.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+using Xunit;
+
+namespace Woodstar.FunctionalTests;
+
+[CollectionDefinition("Database")]
+public class DatabaseService : ICollectionFixture<DatabaseService>
+{
+    public const string EndPoint = "127.0.0.1:1433";
+    public const string Username = "<USERNAME>";
+    public const string Password = "<PASSWORD>";
```
Comment (AndriySvyryd) [inline Woodstar/Woodstar.FunctionalTests/DatabaseService.cs:12]:
```suggestion
    public const string Password = "PLACEHOLDER";
```

**Pull request merged**